cmd/release-controller/types.go

```diff
@@ -251,6 +277,10 @@ const (
 
 	releaseAnnotationFromTag = "release.openshift.io/from-tag"
 	releaseAnnotationToTag   = "release.openshift.io/tag"
+
+	releaseAnnotationFromImageStream = "release.openshift.io/from-image-stream"
+	releaseAnnotationFromRelease = "release.openshift.io/from-release"
+	releaseUpgradeJob = "release-openshift-origin-installer-e2e-aws-upgrade"
```

Comment:
Remove this constant if you aren't using it (and this should be an input from a stream, not a constant)